### DISCUSSION TITLE
Custom fonts in pdf invoice template

### DISCUSSION THREAD

**Discussion by maurice-kpw** [kimai/kimai#3367] [Questions & Answers] [answered]
Hello! 

I want to use our corporate font in the PDF-invoice. I tried importing the google font via style (with @import and with a local font, placed in public) but that didn't work. My best guess is it has to be registered somewhere within mPDF? But I have no idea how to do so. Is there a way to define a custom (google) font in the twig-template? 

Thank you very much! :)

– Maurice

**Comment by kevinpapst** [comment, marked as answer]
~~Sorry, don't know. Read MPDF documentation. Fonts seems to be a bigger issue.
If you find a solution, please share it! I would be interested as well to know how it works.~~

The solution is now documented at https://www.kimai.org/documentation/pdf-templates.html#fonts

**Comment by maurice-kpw** [comment]
Thanks for getting back to me so quickly! I moved on to creating a word template with the right font etc. :( So sadly I don't have a solution for this problem. But maybe someone else has :)

**Comment by LiaraAlis** [comment]
With my small changes in PR #3509 it is possible to add custom fonts.
Maybe @kevinpapst can merge this PR. 😊

**Comment by Boulino** [comment]
_Apologies, I accidentally deleted my comment instead of editing it. The original comment basically said the same thing, so I hope this helps someone._ 
I found this discussion through Google, and for anyone wondering where to find it in 2025, it is now located in the official documentation here:
https://www.kimai.org/documentation/pdf-templates.html#custom-fonts
